### PR TITLE
修改mod解析失败问题

### PR DESCRIPTION
**Pull request by Jackson-soft**

### CODE REVIEW

File: java_exception.go [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 package hessian
 
-import "github.com/dubbogo/hessian2/java_exception"
+import "github.com/apache/dubbo-go-hessian2/java_exception"
```
Comment (AlexStocks) [inline java_exception.go]: import (
)
pls.

File: go.mod [hunk 2]
```diff
@@ -1,4 +1,6 @@
-module github.com/dubbogo/hessian2
+module github.com/apache/dubbo-go-hessian2
```
Comment (wongoo) [inline go.mod]: the module name is already fixed in master branch, there is a conflict, pls fix it

File: go.mod [hunk 3]
```diff
@@ -1,4 +1,6 @@
...
+
+go 1.13
```
Comment (wongoo) [inline go.mod]: remove the version required

**Comment by wongoo** [comment]
@Jackson-soft  there is no change for this pr now, will u commit some other changes?

**Comment by wongoo** [comment]
@Jackson-soft  thanks for this pr, any new pr is welcome in the future.

**Pull request closed without merging**